internal/openurl/openurl_test.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2019 The Go Cloud Development Kit Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package openurl_test
+
+import (
+	"testing"
+
+	"gocloud.dev/internal/openurl"
+)
+
+func TestSchemeMap(t *testing.T) {
+	const foo, bar = "foo value", "bar value"
+
+	tests := []struct {
+		URL     string
```

Comment:
Don't export these fields. It doesn't really matter, but it's jarring to read, since we never do export them.

Reply:
Done.